internal/jobs/queue/queue_interface.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 type Enqueuer interface {
-	Enqueue(ctx context.Context, jobs ...dejq.PendingJob) error
+	Enqueue(ctx context.Context, jobs ...dejq.PendingJob) ([]string, error)
 }
```

Comment:
This change is needed because dejq v1.4 changed its API. And yes, I did a breaking change in a minor release, there are no other users of dejq than me :-)